content/en/tracing/trace_collection/trace_context_propagation/_index.md

```diff
@@ -16,17 +16,276 @@ further_reading:
+| [W3C Trace Context][2] | `tracecontext`      |
+| [B3 Single][3]         | `B3 single header`  |
+| [B3 Multi][4]          | `b3`           |
+| None                   | `none`              |
+
+[2]: https://www.w3.org/TR/trace-context/
+[3]: https://github.com/openzipkin/b3-propagation#single-header
+[4]: https://github.com/openzipkin/b3-propagation#multiple-headers
+[5]: https://docs.aws.amazon.com/xray/latest/devguide/aws-xray.html#xray-concepts-tracingheader
+
+{{% /tab %}}
+
+{{% tab "NodeJS" %}}
+
+### Supported propagators
+The Datadog NodeJS SDK maintains several propagators for passing trace context information in different formats:
+
+| Propagator             | Configuration Value |
+|------------------------|---------------------|
+| Datadog                | `datadog`           |
+| [W3C Trace Context][2] | `tracecontext`      |
+| [B3 Single][3]         | `B3 single header`  |
+| [B3 Multi][4]          | `b3multi`           |
+|                        | `B3` (deprecated)   |
+| None                   | `none`              |
+
+[2]: https://www.w3.org/TR/trace-context/
+[3]: https://github.com/openzipkin/b3-propagation#single-header
+[4]: https://github.com/openzipkin/b3-propagation#multiple-headers
+[5]: https://docs.aws.amazon.com/xray/latest/devguide/aws-xray.html#xray-concepts-tracingheader
+
+{{% /tab %}}
+
+{{% tab "PHP" %}}
+
+### Supported propagators
+The Datadog PHP SDK maintains several propagators for passing trace context information in different formats:
+
+| Propagator             | Configuration Value |
+|------------------------|---------------------|
+| Datadog                | `datadog`           |
+| [W3C Trace Context][2] | `tracecontext`      |
+| [B3 Single][3]         | `B3 single header`  |
+| [B3 Multi][4]          | `b3multi`           |
+|                        | `B3` (deprecated)   |
+| None                   | `none`              |
+
+### Additional use cases
+For use cases specific to the Datadog PHP SDK, see the [PHP Trace Context Propagation][5] page.
+
+[2]: https://www.w3.org/TR/trace-context/
+[3]: https://github.com/openzipkin/b3-propagation#single-header
+[4]: https://github.com/openzipkin/b3-propagation#multiple-headers
+[5]: /tracing/trace_collection/trace_context_propagation/php
+
+{{% /tab %}}
+
+{{% tab "C++" %}}
+
+### Supported propagators
+The Datadog C++ SDK maintains several propagators for passing trace context information in different formats:
+
+| Propagator             | Configuration Value |
+|------------------------|---------------------|
+| Datadog                | `datadog`           |
+| [W3C Trace Context][2] | `tracecontext`      |
+| [B3 Multi][4]          | `b3`                |
+|                        | `b3multi`           |
+| None                   | `none`              |
+
+### Additional configuration
+In addition to the environment variable configuration, you can also update the propagators in code by using `Datadog.configure`:
```

Comment:
```suggestion
In addition to the environment variable configuration, you can also update the propagators in code:
```

Reply:
Fixed in 5537d5bcfd719ca17485bd10969911540378ae72